generated/java/gapic-google-cloud-error-reporting-v1beta1/src/main/java/com/google/cloud/errorreporting/spi/v1beta1/ErrorGroupServiceClient.java

```diff
@@ -87,7 +87,7 @@
  * </pre>
  */
 @Generated("by GAPIC")
-@ExperimentalApi
+@BetaApi
```

Comment:
Could you do a separate regeneration of the Java clients so that this noise isn't mixed in with your restructuring?